free-courses-en.md

```diff
@@ -113,10 +112,6 @@
    
 
 
-### Clojure
```

Comment:
As there must be **two** blank lines in 2 section, You need to delete blank line **113** and **114** for passing machine checking</br>
example:
```markdown
[...]
* [An Awesome Book](http://example.com/example.html)
                                (blank line)
                                (blank line)
### Example
                                (blank line)
* [Another Awesome Book](http://example.com/book.html)
* [Some Other Book](http://example.com/other.html)
```